From: Arnaud Ferraris <arnaud.ferraris@collabora.com>
Date: Mon, 7 Mar 2022 13:59:41 +0100
Subject: arm64: dts: rk3399-pinephone-pro: add "near level" to proximity
 sensor

This will ease the task of userspace software without requiring
additional configuration files.

Signed-off-by: Arnaud Ferraris <arnaud.ferraris@collabora.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index 45de267..ea49a37 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -975,6 +975,7 @@ light-sensor@48 {
 		reg = <0x48>;
 		interrupt-parent = <&gpio4>;
 		interrupts = <RK_PD3 IRQ_TYPE_EDGE_FALLING>;
+		proximity-near-level = <32>;
 		vdd-supply = <&vcc_3v0>;
 		leda-supply = <&vcc_3v0>;
 	};
